src/Dapper.FSharp/IDbConnection.fs

```diff
@@ -5,9 +5,10 @@ open Dapper
 
 type LogFn = string * Map<string, obj> -> unit
 
-let query1<'a> (this:IDbConnection) trans timeout (logFunction:LogFn option) (query, pars) =
+let query1<'a> (this:IDbConnection) trans timeout cancellationToken (logFunction:LogFn option) (query, pars) =
     if logFunction.IsSome then (query, pars) |> logFunction.Value
-    this.QueryAsync<'a>(query, pars, transaction = Option.toObj trans, commandTimeout = Option.toNullable timeout)
+    CommandDefinition(query, pars, ?transaction = trans, ?commandTimeout = timeout, ?cancellationToken = cancellationToken)
```

Comment:
`CommandDefinition` is how you pass along a `CancellationToken` to `QueryAsync`.

Additionally made changes like `transaction = Option.toObj` -> `?transaction = trans` as this is the same but in F# syntax sugar way.

Reply:
Nice! I didn't know about it. Since this is all internal implementation I am totally ok with anything that makes it better (and pass the tests).